e2e/cli-e2e/tests/__snapshots__/help.spec.ts.snap

```diff
@@ -4,7 +4,10 @@ exports[`CLI help > should print help 1`] = `
 "[1mCode PushUp CLI[22m
 
 Commands:
-  code-pushup          Run Plugins and collect results                 [default]
+  code-pushup          Autorun executes the collect and upload command after ano
+                       ther                                            [default]
+  code-pushup autorun  Autorun executes the collect and upload command after ano
+                       ther
```

Comment:
The line break here is unfortunate :grimacing:

I guess we need to keep the descriptions short. How about this?

```suggestion
  code-pushup autorun  Shortcut for running collect followed by upload
```

Reply:
The description hasn't been changed, so marking as unresolved.

My code suggestion is just for demonstration in this case, obviously this has to be changed at source and then updated in the snapshots (with `-u`), not by just accepting the suggestion in GitHub. Would it help if I was clearer when it's a suggestion that can be accepted "as is", or when it's just a diff for illustration (has to changed in multiple places)?

Reply:
missed it!

Reply:
done